src/components/organisms/UiMenu/stories/HasButton.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <UiPopover class="has-button">
+    <UiMenu
+      v-bind="args"
+      role="listbox"
+      :aria-label="ariaLabel"
+      aria-multiselectable="true"
+      :aria-activedescendant="ariaActivedescendant"
+      :aria-busy="ariaBusy"
+    >
+      <template #[`menu-item-${index}`] v-for="(item, index) in itemsToRender">
+        <template v-if="item.hasCustomOption">
+          <UiText
+            tag="span"
+            class="ui-menu-item-button__title">
+            <UiIcon
+              :icon="item.icon"
+              class="ui-button__icon" />
+            <UiText
+              tag="span"
+              class="ui-menu-item-button__label"
+              v-bind="item.labelButtonAttrs">
+              {{ item.label }}
+            </UiText>
+          </UiText>
+          <UiText
+            tag="span"
+            class="ui-menu-item-button__hint">
+            {{ item.hint }}
+          </UiText>
+        </template>
+        <template v-else>
+          <UiCheckbox :aria-label="item.label" disabled>{{ item.label }}</UiCheckbox>
+        </template>
+      </template>
+    </UiMenu>
+  </UiPopover>
+</template>
+
+<script setup lang="ts">
+import {
+  computed,
+  useAttrs,
+  defineOptions,
+} from 'vue';
+import {
+  UiButton,
+  UiCheckbox,
+  UiIcon,
+  UiMenu,
+  UiPopover,
+  UiText,
+} from '@infermedica/component-library';
+
+defineOptions({ inheritAttrs: false });
+const attrs = useAttrs();
+const args = computed(() => (attrs));
```

Comment:
```suggestion
const args = computed(() => (Object.keys(attrs).reduce((object, key)=>{
if(key !== items) {
return {
...object,
[key]: args[key]
}
}
}, {})));
```
you bind `args` in line:4 where you use items instead renderItems. Please remove items from args 🥺